<launch>
    <node name="uvp6_driver" pkg="uvp6" type="uvp6_driver.py">
        <param name="port" value="/dev/ttyUSB0"/>
        <param name="baudrate" value="9600"/>
    </node>
</launch>
